build.sh

```diff
@@ -100,5 +100,14 @@ for dir in ${dirs}; do
     fi
   fi
 
+  if [[ -v TEST_OPENSHIFT_MODE ]]; then
+    if [[ -x test/run-openshift ]]; then
+      VERSION=$dir IMAGE_NAME=${IMAGE_NAME} test/run-openshift
+    else
+      echo "-> OpenShift tests are not present, skipping"
+    fi
+  fi
+
+
```

Comment:
> What I dislike about build.sh script is that it does both building and testing.

By that I mean that usually this is solved a declarative way;  by separate targets in Makefile and dependencies between targets.